phpdotnet/phd/Package/PHP/Web.php

```diff
@@ -116,13 +117,22 @@ public function header($id) {
             if (!isset($written_toc[$filename])) {
                 $written_toc[$filename] = true;
 
-                $toc = array();
+                $toc = $toc_deprecated = array();
 
                 foreach($siblings as $sid) {
-                    $toc[] = array(
-                        Format::getFilename($sid).$ext,
-                        Format::getShortDescription($sid),
-                    );
+                    $sibling_short_desc = Format::getShortDescription($sid);
+
+                    if (!isset($this->deprecated[$sibling_short_desc])) {
+                        $toc[] = array(
+                            Format::getFilename($sid).$ext,
+                            $sibling_short_desc,
+                        );
+                    } else {
+                        $toc_deprecated[] = array(
+                            Format::getFilename($sid).$ext,
+                            $sibling_short_desc,
+                        );
+                    }
```

Comment:
To avoid future confusion, this should likely be
$entry = array(.....);
and the if (deprecated) { $toc_deprecated[] = $entry; } else { $toc[] = $entry; }